fe/fe-core/src/main/java/com/starrocks/connector/hive/ConnectorTableMetadataProcessor.java

```diff
@@ -52,6 +66,38 @@ protected void runAfterCatalogReady() {
         }
 
         refreshRegisteredTable();
+        refreshCatalogTable();
+    }
+
+    private void refreshCatalogTable() {
+        MetadataMgr metadataMgr = GlobalStateMgr.getCurrentState().getMetadataMgr();
+        List<String> catalogNames = Lists.newArrayList(cacheUpdateProcessors.keySet());
+        for (String catalogName : catalogNames) {
+            CacheUpdateProcessor updateProcessor = cacheUpdateProcessors.get(catalogName);
+            if (updateProcessor == null) {
+                LOG.error("Failed to get cacheUpdateProcessor by catalog {}.", catalogName);
+                continue;
+            }
+
+            for (HiveTableName cachedTableName : updateProcessor.getCachedTableNames()) {
+                String dbName = cachedTableName.getDatabaseName();
+                String tableName = cachedTableName.getTableName();
+                Table table;
+                try {
+                    table = metadataMgr.getTable(catalogName, dbName, tableName);
+                } catch (Exception e) {
+                    LOG.warn("can't get table of {}.{}.{}", catalogName, dbName, tableName);
+                    continue;
+                }
+                if (table == null) {
+                    LOG.warn("{}.{}.{} not exist", catalogName, dbName, tableName);
+                    continue;
+                }
+                updateProcessor.refreshTableMetaStoreInfo(dbName, table, true);
```

Comment:
I think this part should be wrapped in catch. If there are some exceptions, we could get error message from fe.log.